modules/ROOT/pages/api-groups-landing-page.adoc

```diff
@@ -0,0 +1,94 @@
+= API Groups
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Using API Groups enables you to bundle your APIs and resources to solve specific user needs. Instead of using individual resources and APIs from a list, the users can request access to and use these in a package that solves a specific problem for them. 
+
+When you create an API Group, you can customize instances of that group with different SLAs and rate limit conditions based on the problem the instance must solve. You can then publish the API Group to Anypoint Exchange so that users can subscribe to the package.
+
+Because each API Group instance can include several API instances, and each instance can be included in multiple API Group instance solutions, package customization options are limitless. 
+
+== How API Groups Works
+
+As an API Group Administrator, you create an API Group for a specific organization and environment. API Groups are versioned, like the APIs contained within them, to more easily incorporate updates.  
+
+Each API Group can have multiple instances and each API Group instance can have multiple API instance. These API Group instances can exists in different environments, such as production and sandbox. 
+
+The following graphic illustrates how you can implement API Groups in different environments:
+image:api-groups-architecture.png[API Group Hierarchy]
+
+In this example, we have two business groups: `ORG A` and `ORG B`. Each of these business groups has two environments: `Sandbox` and `Production`. `ORG A` has two APIs: `Twitter` and `Facebook`, and one instance of each API in the two environments. `ORG B` has two APIs: `Instagram` and `Snapchat`, and one instance of each API in the two environments.
+
+`ORG A` has one API Group called `Social Media` with one version `V1`. `Social Media V1` has 2 API Group instances:
+
+* API Group instance `QA` in `Sandbox` environment. This instance has 4 API instances: `Facebook v1` (QA) and `Twitter v1` (QA) from `ORG A` and `Instagram v1` (QA) and `Snapchat v1` (QA) from `ORG B`.
+* API Group instance `PROD` in `Production` environment. This instance has 4 API instances: `Facebook v1` (PROD) and `Twitter v1` (PROD) from `ORG A` and `Instagram v1` (PROD) and `Snapchat v1` (PROD) from `ORG B`. 
+
+The API Group instance `QA` is published and an asset called `API Group social media V1` is created. If the API Group instance `PROD` is published, no asset will be created. In this case, `PROD` will only be linked to the existing asset because the API Group `Social Media` and the version `v1` are the same.
```

Comment:
```suggestion
The API Group instance `QA` is published and an asset called `API Group social media V1` is created. If the API Group instance `PROD` is published, no asset is created. In this case, `PROD` is linked only to the existing asset because the API Group `Social Media` and the version `v1` are the same.
```

Reply:
Done! Thanks!